db/migrate/20150729041232_add_piece_type_to_pieces.rb

```diff
@@ -0,0 +1,6 @@
+class AddPieceTypeToPieces < ActiveRecord::Migration
+
+  def change
+  	rename_column :pieces, :type, :piece_type
```

Comment:
@mscoutermarsh add_column would not work. It kept throwing an error. Should it be add_column and we missed something else, or is rename column correct?